### ISSUE TITLE
Got `subprocess.TimeoutExpired` when run `./breeze --python 3.8 buildl-docs` on Apple M1 chip

### ISSUE THREAD

**Issue by zkan** [apache/airflow#17310]
**Apache Airflow version**: 2.2.0.dev0
**Python version**: 3.8
**Docker image**: ghcr.io/apache/airflow-main-python3.8-ci-v2

**Environment**:

- **OS** (e.g. from /etc/os-release): macOS Big Sur (11.4)
- **Kernel** (e.g. `uname -a`): Darwin Kans-Mac-mini.local 20.5.0 Darwin Kernel Version 20.5.0: Sat May  8 05:10:31 PDT 2021; root:xnu-7195.121.3~9/RELEASE_ARM64_T8101 arm64
- **Others**: Docker Desktop for Mac (3.5.2.18)

**What happened**:

Got the error below when I run the command `./breeze --python 3.8 build-docs`.

```
subprocess.TimeoutExpired: Command '['/opt/airflow/docs/exts/docs_build/run_patched_sphinx.py', '-T', '--color', '-b', 'html', '-d', '/opt/airflow/docs/_doctrees/docs/apache-airflow', '-c', '/opt/airflow/docs', '-w', '/opt/airflow/docs/_build/docs/apache-airflow/latest/warning-build-apache-airflow.log', '/opt/airflow/docs/apache-airflow', '/opt/airflow/docs/_build/docs/apache-airflow/latest']' timed out after 479.999705667 seconds
```

I think this is not related to the memory for Docker Engine since I've tried to increase it to 15GB.

**What you expected to happen**:

The build should be successful.

**How to reproduce it**:

1. Run `./breeze --python 3.8 build-docs` on Apple M1 chip.
2. Got the `TimeoutExpired` error.

**Anything else we need to know**:

When I tried on my MacBook Pro 15", it works just fine. Here is the spec:

- **OS** (e.g. from /etc/os-release): macOS Big Sur (11.5)
- **Kernel** (e.g. `uname -a`): Darwin BearCamp.local 20.6.0 Darwin Kernel Version 20.6.0: Wed Jun 23 00:26:31 PDT 2021; root:xnu-7195.141.2~5/RELEASE_X86_64 x86_64
- **Others**: Docker Desktop for Mac (3.5.0.5309)

Please see the entire log below.

<details><summary>Entire Log</summary>

<pre><code>➜  airflow git:(main) ./breeze --python 3.8 build-docs
Python version: 3.8

28afd8a9b443a3a994b1ba9b0d1408433aa5df5ab2695515013672e12cc79b66

   CI image.

   Branch name:             main
   Docker image:            ghcr.io/apache/airflow-main-python3.8-ci-v2

   Airflow source version:  2.2.0.dev0
   Python version:          3.8
   Backend:                 sqlite

Memory available for Docker: 7.8G
CPUS available for Docker: 4
Disk space available for Docker: 86G

Run build docs

WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
#################### Available packages ####################
 - apache-airflow
 - apache-airflow-providers
 - apache-airflow-providers-airbyte
 - apache-airflow-providers-amazon
 - apache-airflow-providers-apache-beam
 - apache-airflow-providers-apache-cassandra
 - apache-airflow-providers-apache-drill
 - apache-airflow-providers-apache-druid
 - apache-airflow-providers-apache-hdfs
 - apache-airflow-providers-apache-hive
 - apache-airflow-providers-apache-kylin
 - apache-airflow-providers-apache-livy
 - apache-airflow-providers-apache-pig
 - apache-airflow-providers-apache-pinot
 - apache-airflow-providers-apache-spark
 - apache-airflow-providers-apache-sqoop
 - apache-airflow-providers-asana
 - apache-airflow-providers-celery
 - apache-airflow-providers-cloudant
 - apache-airflow-providers-cncf-kubernetes
 - apache-airflow-providers-databricks
 - apache-airflow-providers-datadog
 - apache-airflow-providers-dingding
 - apache-airflow-providers-discord
 - apache-airflow-providers-docker
 - apache-airflow-providers-elasticsearch
 - apache-airflow-providers-exasol
 - apache-airflow-providers-facebook
 - apache-airflow-providers-ftp
 - apache-airflow-providers-google
 - apache-airflow-providers-grpc
 - apache-airflow-providers-hashicorp
 - apache-airflow-providers-http
 - apache-airflow-providers-imap
 - apache-airflow-providers-jdbc
 - apache-airflow-providers-jenkins
 - apache-airflow-providers-jira
 - apache-airflow-providers-microsoft-azure
 - apache-airflow-providers-microsoft-mssql
 - apache-airflow-providers-microsoft-winrm
 - apache-airflow-providers-mongo
 - apache-airflow-providers-mysql
 - apache-airflow-providers-neo4j
 - apache-airflow-providers-odbc
 - apache-airflow-providers-openfaas
 - apache-airflow-providers-opsgenie
 - apache-airflow-providers-oracle
 - apache-airflow-providers-pagerduty
 - apache-airflow-providers-papermill
 - apache-airflow-providers-plexus
 - apache-airflow-providers-postgres
 - apache-airflow-providers-presto
 - apache-airflow-providers-qubole
 - apache-airflow-providers-redis
 - apache-airflow-providers-salesforce
 - apache-airflow-providers-samba
 - apache-airflow-providers-segment
 - apache-airflow-providers-sendgrid
 - apache-airflow-providers-sftp
 - apache-airflow-providers-singularity
 - apache-airflow-providers-slack
 - apache-airflow-providers-snowflake
 - apache-airflow-providers-sqlite
 - apache-airflow-providers-ssh
 - apache-airflow-providers-tableau
 - apache-airflow-providers-telegram
 - apache-airflow-providers-trino
 - apache-airflow-providers-vertica
 - apache-airflow-providers-yandex
 - apache-airflow-providers-zendesk
 - docker-stack
 - helm-chart
#################### Fetching inventories ####################
To download 106 inventorie(s)
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-apache-druid/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-apache-hdfs/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-apache-cassandra/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-apache-drill/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-airbyte/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-apache-beam/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-amazon/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-apache-livy/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-apache-hive/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-apache-kylin/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-apache-pig/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-apache-pinot/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-apache-spark/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-asana/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-apache-sqoop/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-celery/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-datadog/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-cloudant/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-databricks/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-cncf-kubernetes/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-dingding/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-discord/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-docker/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-elasticsearch/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-facebook/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-exasol/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-ftp/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-grpc/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-http/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-imap/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-jenkins/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-hashicorp/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-google/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-jdbc/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-jira/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-microsoft-azure/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-microsoft-mssql/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-microsoft-winrm/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-mongo/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-mysql/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-neo4j/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-odbc/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-opsgenie/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-openfaas/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-oracle/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-papermill/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-pagerduty/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-postgres/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-plexus/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-presto/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-qubole/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-redis/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-salesforce/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-samba/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-segment/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-sftp/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-singularity/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-sendgrid/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-slack/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-snowflake/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-sqlite/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-ssh/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-tableau/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-telegram/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-trino/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-vertica/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-yandex/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-zendesk/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/helm-chart/latest/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/docker-stack/objects.inv
Fetched inventory: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow/latest/objects.inv
Fetched inventory: https://docs.python.org/3/objects.inv
Fetched inventory: https://googleapis.dev/python/google-api-core/latest/objects.inv
Fetched inventory: https://docker-py.readthedocs.io/en/stable/objects.inv
Fetched inventory: https://hdfscli.readthedocs.io/en/latest/objects.inv
Fetched inventory: https://jinja.palletsprojects.com/en/2.11.x/objects.inv
Fetched inventory: https://pymongo.readthedocs.io/en/3.11.3/objects.inv
Fetched inventory: https://boto3.amazonaws.com/v1/documentation/api/latest/objects.inv
Fetched inventory: https://docs.celeryproject.org/en/stable/objects.inv
Fetched inventory: https://googleapis.dev/python/bigquerystorage/latest/objects.inv
Fetched inventory: https://googleapis.dev/python/automl/latest/objects.inv
Fetched inventory: https://googleapis.dev/python/bigquery/latest/objects.inv
Fetched inventory: https://googleapis.dev/python/bigquerydatatransfer/latest/objects.inv
Fetched inventory: https://googleapis.dev/python/bigtable/latest/objects.inv
Fetched inventory: https://googleapis.dev/python/dlp/latest/objects.inv
Fetched inventory: https://googleapis.dev/python/container/latest/objects.inv
Fetched inventory: https://googleapis.dev/python/datastore/latest/objects.inv
Fetched inventory: https://googleapis.dev/python/cloudkms/latest/objects.inv
Fetched inventory: https://googleapis.dev/python/language/latest/objects.inv
Fetched inventory: https://googleapis.dev/python/datacatalog/latest/objects.inv
Fetched inventory: https://requests.readthedocs.io/en/master/objects.inv
Fetched inventory: https://googleapis.dev/python/google-cloud-core/latest/objects.inv
Fetched inventory: https://googleapis.dev/python/pubsub/latest/objects.inv
Fetched inventory: https://googleapis.dev/python/storage/latest/objects.inv
Fetched inventory: https://googleapis.dev/python/redis/latest/objects.inv
Fetched inventory: https://googleapis.dev/python/texttospeech/latest/objects.inv
Fetched inventory: https://googleapis.dev/python/monitoring/latest/objects.inv
Fetched inventory: https://googleapis.dev/python/spanner/latest/objects.inv
Fetched inventory: https://googleapis.dev/python/speech/latest/objects.inv
Fetched inventory: https://googleapis.dev/python/videointelligence/latest/objects.inv
Fetched inventory: https://googleapis.dev/python/cloudtasks/latest/objects.inv
Fetched inventory: https://pandas.pydata.org/pandas-docs/stable/objects.inv
Fetched inventory: https://googleapis.dev/python/vision/latest/objects.inv
Fetched inventory: https://googleapis.dev/python/translation/latest/objects.inv
Fetched inventory: https://docs.sqlalchemy.org/en/latest/objects.inv
Result: 106 success, 0 failed
#################### Documentation will be built for 72 package(s) with 4 parallel jobs ####################
1. apache-airflow
2. apache-airflow-providers-airbyte
3. apache-airflow-providers-amazon
4. apache-airflow-providers-apache-beam
5. apache-airflow-providers-apache-cassandra
6. apache-airflow-providers-apache-drill
7. apache-airflow-providers-apache-druid
8. apache-airflow-providers-apache-hdfs
9. apache-airflow-providers-apache-hive
10. apache-airflow-providers-apache-kylin
11. apache-airflow-providers-apache-livy
12. apache-airflow-providers-apache-pig
13. apache-airflow-providers-apache-pinot
14. apache-airflow-providers-apache-spark
15. apache-airflow-providers-apache-sqoop
16. apache-airflow-providers-asana
17. apache-airflow-providers-celery
18. apache-airflow-providers-cloudant
19. apache-airflow-providers-cncf-kubernetes
20. apache-airflow-providers-databricks
21. apache-airflow-providers-datadog
22. apache-airflow-providers-dingding
23. apache-airflow-providers-discord
24. apache-airflow-providers-docker
25. apache-airflow-providers-elasticsearch
26. apache-airflow-providers-exasol
27. apache-airflow-providers-facebook
28. apache-airflow-providers-ftp
29. apache-airflow-providers-google
30. apache-airflow-providers-grpc
31. apache-airflow-providers-hashicorp
32. apache-airflow-providers-http
33. apache-airflow-providers-imap
34. apache-airflow-providers-jdbc
35. apache-airflow-providers-jenkins
36. apache-airflow-providers-jira
37. apache-airflow-providers-microsoft-azure
38. apache-airflow-providers-microsoft-mssql
39. apache-airflow-providers-microsoft-winrm
40. apache-airflow-providers-mongo
41. apache-airflow-providers-mysql
42. apache-airflow-providers-neo4j
43. apache-airflow-providers-odbc
44. apache-airflow-providers-openfaas
45. apache-airflow-providers-opsgenie
46. apache-airflow-providers-oracle
47. apache-airflow-providers-pagerduty
48. apache-airflow-providers-papermill
49. apache-airflow-providers-plexus
50. apache-airflow-providers-postgres
51. apache-airflow-providers-presto
52. apache-airflow-providers-qubole
53. apache-airflow-providers-redis
54. apache-airflow-providers-salesforce
55. apache-airflow-providers-samba
56. apache-airflow-providers-segment
57. apache-airflow-providers-sendgrid
58. apache-airflow-providers-sftp
59. apache-airflow-providers-singularity
60. apache-airflow-providers-slack
61. apache-airflow-providers-snowflake
62. apache-airflow-providers-sqlite
63. apache-airflow-providers-ssh
64. apache-airflow-providers-tableau
65. apache-airflow-providers-telegram
66. apache-airflow-providers-trino
67. apache-airflow-providers-vertica
68. apache-airflow-providers-yandex
69. apache-airflow-providers-zendesk
70. apache-airflow-providers
71. helm-chart
72. docker-stack
#################### Cleaning documentation files ####################
apache-airflow                                              : Cleaning files
apache-airflow-providers-airbyte                            : Cleaning files
apache-airflow-providers-amazon                             : Cleaning files
apache-airflow-providers-apache-beam                        : Cleaning files
apache-airflow-providers-apache-cassandra                   : Cleaning files
apache-airflow-providers-apache-drill                       : Cleaning files
apache-airflow-providers-apache-druid                       : Cleaning files
apache-airflow-providers-apache-hdfs                        : Cleaning files
apache-airflow-providers-apache-hive                        : Cleaning files
apache-airflow-providers-apache-kylin                       : Cleaning files
apache-airflow-providers-apache-livy                        : Cleaning files
apache-airflow-providers-apache-pig                         : Cleaning files
apache-airflow-providers-apache-pinot                       : Cleaning files
apache-airflow-providers-apache-spark                       : Cleaning files
apache-airflow-providers-apache-sqoop                       : Cleaning files
apache-airflow-providers-asana                              : Cleaning files
apache-airflow-providers-celery                             : Cleaning files
apache-airflow-providers-cloudant                           : Cleaning files
apache-airflow-providers-cncf-kubernetes                    : Cleaning files
apache-airflow-providers-databricks                         : Cleaning files
apache-airflow-providers-datadog                            : Cleaning files
apache-airflow-providers-dingding                           : Cleaning files
apache-airflow-providers-discord                            : Cleaning files
apache-airflow-providers-docker                             : Cleaning files
apache-airflow-providers-elasticsearch                      : Cleaning files
apache-airflow-providers-exasol                             : Cleaning files
apache-airflow-providers-facebook                           : Cleaning files
apache-airflow-providers-ftp                                : Cleaning files
apache-airflow-providers-google                             : Cleaning files
apache-airflow-providers-grpc                               : Cleaning files
apache-airflow-providers-hashicorp                          : Cleaning files
apache-airflow-providers-http                               : Cleaning files
apache-airflow-providers-imap                               : Cleaning files
apache-airflow-providers-jdbc                               : Cleaning files
apache-airflow-providers-jenkins                            : Cleaning files
apache-airflow-providers-jira                               : Cleaning files
apache-airflow-providers-microsoft-azure                    : Cleaning files
apache-airflow-providers-microsoft-mssql                    : Cleaning files
apache-airflow-providers-microsoft-winrm                    : Cleaning files
apache-airflow-providers-mongo                              : Cleaning files
apache-airflow-providers-mysql                              : Cleaning files
apache-airflow-providers-neo4j                              : Cleaning files
apache-airflow-providers-odbc                               : Cleaning files
apache-airflow-providers-openfaas                           : Cleaning files
apache-airflow-providers-opsgenie                           : Cleaning files
apache-airflow-providers-oracle                             : Cleaning files
apache-airflow-providers-pagerduty                          : Cleaning files
apache-airflow-providers-papermill                          : Cleaning files
apache-airflow-providers-plexus                             : Cleaning files
apache-airflow-providers-postgres                           : Cleaning files
apache-airflow-providers-presto                             : Cleaning files
apache-airflow-providers-qubole                             : Cleaning files
apache-airflow-providers-redis                              : Cleaning files
apache-airflow-providers-salesforce                         : Cleaning files
apache-airflow-providers-samba                              : Cleaning files
apache-airflow-providers-segment                            : Cleaning files
apache-airflow-providers-sendgrid                           : Cleaning files
apache-airflow-providers-sftp                               : Cleaning files
apache-airflow-providers-singularity                        : Cleaning files
apache-airflow-providers-slack                              : Cleaning files
apache-airflow-providers-snowflake                          : Cleaning files
apache-airflow-providers-sqlite                             : Cleaning files
apache-airflow-providers-ssh                                : Cleaning files
apache-airflow-providers-tableau                            : Cleaning files
apache-airflow-providers-telegram                           : Cleaning files
apache-airflow-providers-trino                              : Cleaning files
apache-airflow-providers-vertica                            : Cleaning files
apache-airflow-providers-yandex                             : Cleaning files
apache-airflow-providers-zendesk                            : Cleaning files
apache-airflow-providers                                    : Cleaning files
helm-chart                                                  : Cleaning files
docker-stack                                                : Cleaning files
#################### Scheduling documentation to build ####################
apache-airflow                                              : Scheduling documentation to build
apache-airflow-providers-airbyte                            : Scheduling documentation to build
apache-airflow-providers-amazon                             : Scheduling documentation to build
apache-airflow-providers-apache-beam                        : Scheduling documentation to build
apache-airflow-providers-apache-cassandra                   : Scheduling documentation to build
apache-airflow-providers-apache-drill                       : Scheduling documentation to build
apache-airflow-providers-apache-druid                       : Scheduling documentation to build
apache-airflow-providers-apache-hdfs                        : Scheduling documentation to build
apache-airflow-providers-apache-hive                        : Scheduling documentation to build
apache-airflow-providers-apache-kylin                       : Scheduling documentation to build
apache-airflow-providers-apache-livy                        : Scheduling documentation to build
apache-airflow-providers-apache-pig                         : Scheduling documentation to build
apache-airflow-providers-apache-pinot                       : Scheduling documentation to build
apache-airflow-providers-apache-spark                       : Scheduling documentation to build
apache-airflow-providers-apache-sqoop                       : Scheduling documentation to build
apache-airflow-providers-asana                              : Scheduling documentation to build
apache-airflow-providers-celery                             : Scheduling documentation to build
apache-airflow-providers-cloudant                           : Scheduling documentation to build
apache-airflow-providers-cncf-kubernetes                    : Scheduling documentation to build
apache-airflow-providers-databricks                         : Scheduling documentation to build
apache-airflow-providers-datadog                            : Scheduling documentation to build
apache-airflow-providers-dingding                           : Scheduling documentation to build
apache-airflow-providers-discord                            : Scheduling documentation to build
apache-airflow-providers-docker                             : Scheduling documentation to build
apache-airflow-providers-elasticsearch                      : Scheduling documentation to build
apache-airflow-providers-exasol                             : Scheduling documentation to build
apache-airflow-providers-facebook                           : Scheduling documentation to build
apache-airflow-providers-ftp                                : Scheduling documentation to build
apache-airflow-providers-google                             : Scheduling documentation to build
apache-airflow-providers-grpc                               : Scheduling documentation to build
apache-airflow-providers-hashicorp                          : Scheduling documentation to build
apache-airflow-providers-http                               : Scheduling documentation to build
apache-airflow-providers-imap                               : Scheduling documentation to build
apache-airflow-providers-jdbc                               : Scheduling documentation to build
apache-airflow-providers-jenkins                            : Scheduling documentation to build
apache-airflow-providers-jira                               : Scheduling documentation to build
apache-airflow-providers-microsoft-azure                    : Scheduling documentation to build
apache-airflow-providers-microsoft-mssql                    : Scheduling documentation to build
apache-airflow-providers-microsoft-winrm                    : Scheduling documentation to build
apache-airflow-providers-mongo                              : Scheduling documentation to build
apache-airflow-providers-mysql                              : Scheduling documentation to build
apache-airflow-providers-neo4j                              : Scheduling documentation to build
apache-airflow-providers-odbc                               : Scheduling documentation to build
apache-airflow-providers-openfaas                           : Scheduling documentation to build
apache-airflow-providers-opsgenie                           : Scheduling documentation to build
apache-airflow-providers-oracle                             : Scheduling documentation to build
apache-airflow-providers-pagerduty                          : Scheduling documentation to build
apache-airflow-providers-papermill                          : Scheduling documentation to build
apache-airflow-providers-plexus                             : Scheduling documentation to build
apache-airflow-providers-postgres                           : Scheduling documentation to build
apache-airflow-providers-presto                             : Scheduling documentation to build
apache-airflow-providers-qubole                             : Scheduling documentation to build
apache-airflow-providers-redis                              : Scheduling documentation to build
apache-airflow-providers-salesforce                         : Scheduling documentation to build
apache-airflow-providers-samba                              : Scheduling documentation to build
apache-airflow-providers-segment                            : Scheduling documentation to build
apache-airflow-providers-sendgrid                           : Scheduling documentation to build
apache-airflow-providers-sftp                               : Scheduling documentation to build
apache-airflow-providers-singularity                        : Scheduling documentation to build
apache-airflow-providers-slack                              : Scheduling documentation to build
apache-airflow-providers-snowflake                          : Scheduling documentation to build
apache-airflow-providers-sqlite                             : Scheduling documentation to build
apache-airflow-providers-ssh                                : Scheduling documentation to build
apache-airflow-providers-tableau                            : Scheduling documentation to build
apache-airflow-providers-telegram                           : Scheduling documentation to build
apache-airflow-providers-trino                              : Scheduling documentation to build
apache-airflow-providers-vertica                            : Scheduling documentation to build
apache-airflow-providers-yandex                             : Scheduling documentation to build
apache-airflow-providers-zendesk                            : Scheduling documentation to build
apache-airflow-providers                                    : Scheduling documentation to build
helm-chart                                                  : Scheduling documentation to build
docker-stack                                                : Scheduling documentation to build
#################### Running docs building ####################

apache-airflow                                              : Building documentation
apache-airflow-providers-apache-drill                       : Building documentation
apache-airflow-providers-apache-livy                        : Building documentation
apache-airflow-providers-asana                              : Building documentation
apache-airflow                                              : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-apache-drill                       : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-apache-livy                        : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-asana                              : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-asana                              : Finished docs building successfully
apache-airflow-providers-celery                             : Building documentation
apache-airflow-providers-celery                             : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-apache-drill                       : Finished docs building successfully
apache-airflow-providers-apache-druid                       : Building documentation
apache-airflow-providers-apache-druid                       : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-apache-livy                        : Finished docs building successfully
apache-airflow-providers-apache-pig                         : Building documentation
apache-airflow-providers-apache-pig                         : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-celery                             : Finished docs building successfully
apache-airflow-providers-cloudant                           : Building documentation
apache-airflow-providers-cloudant                           : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-apache-pig                         : Finished docs building successfully
apache-airflow-providers-apache-pinot                       : Building documentation
apache-airflow-providers-apache-pinot                       : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-apache-druid                       : Finished docs building successfully
apache-airflow-providers-apache-hdfs                        : Building documentation
apache-airflow-providers-apache-hdfs                        : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-cloudant                           : Finished docs building successfully
apache-airflow-providers-cncf-kubernetes                    : Building documentation
apache-airflow-providers-cncf-kubernetes                    : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-apache-pinot                       : Finished docs building successfully
apache-airflow-providers-apache-spark                       : Building documentation
apache-airflow-providers-apache-spark                       : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-apache-hdfs                        : Finished docs building successfully
apache-airflow-providers-apache-hive                        : Building documentation
apache-airflow-providers-apache-hive                        : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-apache-spark                       : Finished docs building successfully
apache-airflow-providers-apache-sqoop                       : Building documentation
apache-airflow-providers-apache-sqoop                       : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-cncf-kubernetes                    : Finished docs building successfully
apache-airflow-providers-databricks                         : Building documentation
apache-airflow-providers-databricks                         : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-apache-sqoop                       : Finished docs building successfully
apache-airflow-providers-datadog                            : Building documentation
apache-airflow-providers-datadog                            : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-databricks                         : Finished docs building successfully
apache-airflow-providers-exasol                             : Building documentation
apache-airflow-providers-exasol                             : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-apache-hive                        : Finished docs building successfully
apache-airflow-providers-apache-kylin                       : Building documentation
apache-airflow-providers-apache-kylin                       : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-datadog                            : Finished docs building successfully
apache-airflow-providers-dingding                           : Building documentation
apache-airflow-providers-dingding                           : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-exasol                             : Finished docs building successfully
apache-airflow-providers-facebook                           : Building documentation
apache-airflow-providers-facebook                           : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-apache-kylin                       : Finished docs building successfully
apache-airflow-providers-hashicorp                          : Building documentation
apache-airflow-providers-hashicorp                          : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-facebook                           : Finished docs building successfully
apache-airflow-providers-ftp                                : Building documentation
apache-airflow-providers-ftp                                : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-dingding                           : Finished docs building successfully
apache-airflow-providers-discord                            : Building documentation
apache-airflow-providers-discord                            : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-hashicorp                          : Finished docs building successfully
apache-airflow-providers-http                               : Building documentation
apache-airflow-providers-http                               : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-discord                            : Finished docs building successfully
apache-airflow-providers-docker                             : Building documentation
apache-airflow-providers-docker                             : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-ftp                                : Finished docs building successfully
apache-airflow-providers-google                             : Building documentation
apache-airflow-providers-google                             : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-docker                             : Finished docs building successfully
apache-airflow-providers-elasticsearch                      : Building documentation
apache-airflow-providers-elasticsearch                      : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-http                               : Finished docs building successfully
apache-airflow-providers-imap                               : Building documentation
apache-airflow-providers-imap                               : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-elasticsearch                      : Finished docs building successfully
apache-airflow-providers-jira                               : Building documentation
apache-airflow-providers-jira                               : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-imap                               : Finished docs building successfully
apache-airflow-providers-jdbc                               : Building documentation
apache-airflow-providers-jdbc                               : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-jira                               : Finished docs building successfully
apache-airflow-providers-microsoft-azure                    : Building documentation
apache-airflow-providers-microsoft-azure                    : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-jdbc                               : Finished docs building successfully
apache-airflow-providers-jenkins                            : Building documentation
apache-airflow-providers-jenkins                            : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-jenkins                            : Finished docs building successfully
apache-airflow-providers-mysql                              : Building documentation
apache-airflow-providers-mysql                              : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-oracle                             : Building documentation
apache-airflow-providers-oracle                             : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-mysql                              : Finished docs building successfully
apache-airflow-providers-neo4j                              : Building documentation
apache-airflow-providers-neo4j                              : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-oracle                             : Finished docs building successfully
apache-airflow-providers-pagerduty                          : Building documentation
apache-airflow-providers-pagerduty                          : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-microsoft-azure                    : Finished docs building successfully
apache-airflow-providers-microsoft-mssql                    : Building documentation
apache-airflow-providers-microsoft-mssql                    : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-pagerduty                          : Finished docs building successfully
apache-airflow-providers-papermill                          : Building documentation
apache-airflow-providers-papermill                          : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-neo4j                              : Finished docs building successfully
apache-airflow-providers-odbc                               : Building documentation
apache-airflow-providers-odbc                               : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-microsoft-mssql                    : Finished docs building successfully
apache-airflow-providers-microsoft-winrm                    : Building documentation
apache-airflow-providers-microsoft-winrm                    : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-odbc                               : Finished docs building successfully
apache-airflow-providers-openfaas                           : Building documentation
apache-airflow-providers-openfaas                           : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-papermill                          : Finished docs building successfully
apache-airflow-providers-plexus                             : Building documentation
apache-airflow-providers-plexus                             : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-microsoft-winrm                    : Finished docs building successfully
apache-airflow-providers-mongo                              : Building documentation
apache-airflow-providers-mongo                              : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-openfaas                           : Finished docs building successfully
apache-airflow-providers-opsgenie                           : Building documentation
apache-airflow-providers-opsgenie                           : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-plexus                             : Finished docs building successfully
apache-airflow-providers-postgres                           : Building documentation
apache-airflow-providers-postgres                           : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-opsgenie                           : Finished docs building successfully
apache-airflow-providers-presto                             : Building documentation
apache-airflow-providers-presto                             : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-mongo                              : Finished docs building successfully
apache-airflow-providers-segment                            : Building documentation
apache-airflow-providers-segment                            : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-postgres                           : Finished docs building successfully
apache-airflow-providers-snowflake                          : Building documentation
apache-airflow-providers-snowflake                          : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-segment                            : Finished docs building successfully
apache-airflow-providers-sendgrid                           : Building documentation
apache-airflow-providers-sendgrid                           : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-presto                             : Finished docs building successfully
apache-airflow-providers-qubole                             : Building documentation
apache-airflow-providers-qubole                             : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-snowflake                          : Finished docs building successfully
apache-airflow-providers-sqlite                             : Building documentation
apache-airflow-providers-sqlite                             : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-sendgrid                           : Finished docs building successfully
apache-airflow-providers-sftp                               : Building documentation
apache-airflow-providers-sftp                               : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-sqlite                             : Finished docs building successfully
apache-airflow-providers-ssh                                : Building documentation
apache-airflow-providers-ssh                                : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-qubole                             : Finished docs building successfully
apache-airflow-providers-redis                              : Building documentation
apache-airflow-providers-redis                              : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-sftp                               : Finished docs building successfully
apache-airflow-providers-singularity                        : Building documentation
apache-airflow-providers-singularity                        : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-ssh                                : Finished docs building successfully
apache-airflow-providers-tableau                            : Building documentation
apache-airflow-providers-tableau                            : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-singularity                        : Finished docs building successfully
apache-airflow-providers-slack                              : Building documentation
apache-airflow-providers-slack                              : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-redis                              : Finished docs building successfully
apache-airflow-providers-salesforce                         : Building documentation
apache-airflow-providers-salesforce                         : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-trino                              : Building documentation
apache-airflow-providers-trino                              : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-tableau                            : Finished docs building successfully
apache-airflow-providers-telegram                           : Building documentation
apache-airflow-providers-telegram                           : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-slack                              : Finished docs building successfully
helm-chart                                                  : Building documentation
helm-chart                                                  : Running sphinx. The output is hidden until an error occurs.
helm-chart                                                  : Finished docs building successfully
docker-stack                                                : Building documentation
docker-stack                                                : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-telegram                           : Finished docs building successfully
apache-airflow-providers-salesforce                         : Finished docs building successfully
apache-airflow-providers-samba                              : Building documentation
apache-airflow-providers-samba                              : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-trino                              : Finished docs building successfully
apache-airflow-providers-vertica                            : Building documentation
apache-airflow-providers-vertica                            : Running sphinx. The output is hidden until an error occurs.
docker-stack                                                : Finished docs building successfully
apache-airflow-providers-samba                              : Finished docs building successfully
apache-airflow-providers-vertica                            : Finished docs building successfully
apache-airflow-providers-yandex                             : Building documentation
apache-airflow-providers-yandex                             : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-yandex                             : Finished docs building successfully
apache-airflow-providers-zendesk                            : Building documentation
apache-airflow-providers-zendesk                            : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers-zendesk                            : Finished docs building successfully
apache-airflow-providers                                    : Building documentation
apache-airflow-providers                                    : Running sphinx. The output is hidden until an error occurs.
apache-airflow-providers                                    : Finished docs building successfully
multiprocessing.pool.RemoteTraceback:
"""
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/usr/local/lib/python3.8/multiprocessing/pool.py", line 48, in mapstar
    return list(map(*args))
  File "/opt/airflow/docs/build_docs.py", line 175, in perform_docs_build_for_single_package
    errors=builder.build_sphinx_docs(
  File "/opt/airflow/docs/exts/docs_build/docs_builder.py", line 242, in build_sphinx_docs
    completed_proc = run(
  File "/usr/local/lib/python3.8/subprocess.py", line 495, in run
    stdout, stderr = process.communicate(input, timeout=timeout)
  File "/usr/local/lib/python3.8/subprocess.py", line 1028, in communicate
    stdout, stderr = self._communicate(input, endtime, timeout)
  File "/usr/local/lib/python3.8/subprocess.py", line 1894, in _communicate
    self.wait(timeout=self._remaining_time(endtime))
  File "/usr/local/lib/python3.8/subprocess.py", line 1083, in wait
    return self._wait(timeout=timeout)
  File "/usr/local/lib/python3.8/subprocess.py", line 1798, in _wait
    raise TimeoutExpired(self.args, timeout)
subprocess.TimeoutExpired: Command '['/opt/airflow/docs/exts/docs_build/run_patched_sphinx.py', '-T', '--color', '-b', 'html', '-d', '/opt/airflow/docs/_doctrees/docs/apache-airflow', '-c', '/opt/airflow/docs', '-w', '/opt/airflow/docs/_build/docs/apache-airflow/latest/warning-build-apache-airflow.log', '/opt/airflow/docs/apache-airflow', '/opt/airflow/docs/_build/docs/apache-airflow/latest']' timed out after 479.999705667 seconds
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/airflow/docs/build_docs.py", line 534, in <module>
    main()
  File "/opt/airflow/docs/build_docs.py", line 475, in main
    package_build_errors, package_spelling_errors = build_docs_for_packages(
  File "/opt/airflow/docs/build_docs.py", line 217, in build_docs_for_packages
    run_in_parallel(
  File "/opt/airflow/docs/build_docs.py", line 289, in run_in_parallel
    run_docs_build_in_parallel(
  File "/opt/airflow/docs/build_docs.py", line 338, in run_docs_build_in_parallel
    result_list = pool.map(perform_docs_build_for_single_package, doc_build_specifications)
  File "/usr/local/lib/python3.8/multiprocessing/pool.py", line 364, in map
    return self._map_async(func, iterable, mapstar, chunksize).get()
  File "/usr/local/lib/python3.8/multiprocessing/pool.py", line 771, in get
    raise self._value
subprocess.TimeoutExpired: Command '['/opt/airflow/docs/exts/docs_build/run_patched_sphinx.py', '-T', '--color', '-b', 'html', '-d', '/opt/airflow/docs/_doctrees/docs/apache-airflow', '-c', '/opt/airflow/docs', '-w', '/opt/airflow/docs/_build/docs/apache-airflow/latest/warning-build-apache-airflow.log', '/opt/airflow/docs/apache-airflow', '/opt/airflow/docs/_build/docs/apache-airflow/latest']' timed out after 479.999705667 seconds
########################################################################################################################
 [IN CONTAINER]   EXITING /opt/airflow/scripts/in_container/run_docs_build.sh WITH EXIT CODE 1
########################################################################################################################

ERROR: The previous step completed with error. Please take a look at output above
</code></pre>
</details>

**Comment by uranusjr** [comment]
Looks like the doc just builds to slowly on the machine. I don’t think there’s much we can do here (except increasing the timeout, which is probably not that meaningful…)

**Comment by potiuk** [comment]
Also you can try to increase the number of CPUs that are available to docker engine. The Build will take advantage of as many CPUS as many you give it to and it will build the documentation in parallel. It looks like the timeout happens because some of the builds were scheduled to run (via paralell) and waited too long, so i f you increase number of CPUs, they will run much faster in general. 

(Please let us know how it works).

Root caus of it is that M1 uses emulation (our Docker image is x86 one) so it will run much slower than native execution.

Also another work is try to run it  locally on Mac - as long as  you have locally instlaled "airflow[doc]" virtualenv, building docs should work with your natively installed python packages - hopefully all sphinx-related libraries can run natively on Mac M1, so you can run it there (it will not take advantage of parallell runs though).

Yet another workaround is that you can just build a single package using for example `./brees build-docs -- --package-filter 'apache-airflow'` to only build airflow documentation.

Taking into account a number of workarounds available and the fact that there is another ticket to implement native M1 docker https://github.com/apache/airflow/issues/15635  I close this ticket as "won't fix".

**Comment by zkan** [comment]
I've tried to increase CPUs to 8 and still got the same issue.

However, the good news is that building a single package `./breeze --python 3.8 build-docs -- --package-filter apache-airflow` works fine. Many thanks! 😄

**Comment by potiuk** [comment]
Glad to hear it helped  :)

**Comment by subkanthi** [comment]
Interestingly there are no issues running breeze on mac os (m1), but getting this error in a Linux VM(Ubuntu) Parallels.
Need to check if docker on Mac is emulating x86.

**Comment by potiuk** [comment]
Never saw this issue on my linux/chromebook/os :(

**Comment by potiuk** [comment]
BTW. Try --python 3.6 :)

**Comment by subkanthi** [comment]
`Step 64/90 : RUN ln -sf /usr/bin/dumb-init /usr/local/bin/dumb-init
 ---> Using cache
 ---> 8bc2b7e35a64
Step 65/90 : COPY airflow/www/yarn.lock airflow/www/package.json ${AIRFLOW_SOURCES}/airflow/www/
 ---> Using cache
 ---> 2101286441e1
Step 66/90 : RUN yarn --cwd airflow/www install --frozen-lockfile --no-cache && yarn cache clean
 ---> Using cache
 ---> 9c2480204136
Step 67/90 : COPY setup.py ${AIRFLOW_SOURCES}/setup.py
 ---> 806de9b1671c
Step 68/90 : COPY setup.cfg ${AIRFLOW_SOURCES}/setup.cfg
 ---> 3b83e41ca3be
Step 69/90 : COPY airflow/__init__.py ${AIRFLOW_SOURCES}/airflow/__init__.py
 ---> c69bf4520461
Step 70/90 : RUN if [[ ${INSTALL_FROM_PYPI} == "true" ]]; then         bash /scripts/docker/install_airflow.sh;     fi
 ---> [Warning] The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
 ---> Running in a890b3ba0c58
standard_init_linux.go:228: exec user process caused: exec format error
The command '/bin/bash -o pipefail -e -u -x -c if [[ ${INSTALL_FROM_PYPI} == "true" ]]; then         bash /scripts/docker/install_airflow.sh;     fi' returned a non-zero code: 1

ERROR: The previous step completed with error. Please take a look at output above 

`

**Comment by potiuk** [comment]
Ah yeah. I believe the Docker Desktop for Mac has fixed the emulation but Parallels on ARM still does not have it :(
